core/domain/exp_domain.py

```diff
@@ -48,6 +48,8 @@
 STATE_PROPERTY_CONTENT = 'content'
 STATE_PROPERTY_SOLICIT_ANSWER_DETAILS = 'solicit_answer_details'
 STATE_PROPERTY_RECORDED_VOICEOVERS = 'recorded_voiceovers'
+STATE_PROPERTY_CONTENT_IDS_TO_AUDIO_TRANSLATIONS_DEPRECATED = (
```

Comment:
Can you please move this blew and add a comment saying it's deprecated in vX of exploration?

Reply:
Done.